src/main/java/com/company/verifytests/CandidatesPage.java

```diff
@@ -0,0 +1,112 @@
+package com.company.verifytests;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class CandidatesPage {
+    public WebDriver driver;
+
+    @FindBy(xpath ="//select[@id='candidateSearch_jobTitle']//option[contains(text(),'AQA')]")
+    private WebElement jobTitleCandidates;
+
+    @FindBy(xpath ="//input[@id='btnAdd']")
+    private WebElement addCandidateButton;
+
```

Comment:
If attributes id or name exists for element, it is best locator. Using xpath is not mistake, so it is ok to leave it as is.

Reply:
Done